cypress/e2e/API/customers.cy.js

```diff
@@ -0,0 +1,167 @@
+/// <reference types="Cypress" />
+
+describe('GET /customers', () => {
+    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
```

Comment:
```suggestion
    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}customers`

```

É uma boa prática deixar uma linha entre a definição de variáveis e outras instruções JavaScript.

A propósito, a descrição da variávle não condiz com seu valor, para condizer, deveria ter a _string_ `/customers` ao final, conforme sugerido.

Dessa forma, evita-se repetir tal _string_ várias vezes por todos os testes de API.